packages/cliqz-home-react/src/TopMessages.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import cliqz from './Cliqz';
+
+class TopMessages extends Component {
+
+  handleClose(id, handler) {    
+    cliqz.freshtab.dismissMessage(id, handler);
+    cliqz.storage.setState((prevState) => {
+      delete prevState.messages[id];
```

Comment:
please don't change `prevState`. react is all about functional programming so state mutations like this are discouraged - besides, I'm not sure but altering `prevState` may have some sideeffects like unnecessary rerender.